### ISSUE TITLE
[Suggestion] Freecam should show camera coords

### ISSUE THREAD

**Issue by JavaNameConvention** [MeteorDevelopment/meteor-client#2098]
### Describe the feature

When toggling freecam it should show camera coords not player coords. 

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client.
(I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested.
(I have checked every suggestion on github)

- [X] This feature is a valid suggestion.
(I have read the FAQs and Guides)

**Comment by MineGame159** [comment]
"it"

**Comment by JavaNameConvention** [comment]
ok ok we get the overall meaning ok

**Comment by MineGame159** [comment]
What? I have no idea what you mean by "it should show camera coords". What is "it".

**Comment by Justgamer101** [comment]
he obviously means he wants the camera coords rendered on his mouse irl

**Comment by mdo992** [comment]
> What? I have no idea what you mean by "it should show camera coords". What is "it".

He probably means that the coordinates should change on the HUD to where ever you are at in freecam.

**Comment by JavaNameConvention** [comment]
Yes, it is like what wurst does where in freecam, when you move around, the coordinates shown in f3 are the camera coordinates. I have yet to go into HUD settings but I think if it is already in the client, it should at least be a more clear setting. Perhaps in the freecam config menu?

**Comment by PeppermentButler** [comment]
I am pretty sure it is not in freecam settings, or in the client at all. Though I have been wanting this addition myself for a bit now :3
